### ISSUE TITLE
Display the alt text below the image

### ISSUE THREAD

**Issue by sitelint** [sitelint/accessible-gallery#3]
+1 feature request: it would be awesome to be able to display the alt text below the image, like adding a data-alt attribute to the accessible_gallery_modal_inner_with_image div so it could be displayed in an ::after element with CSS.

**Comment by sitelint** [comment]
@szilardka The image description should be displayed now based on the image alt text.
